Modules/_sqlite/cursor.c

```diff
@@ -631,7 +631,7 @@ pysqlite_cursor_execute_impl(pysqlite_Cursor *self, PyObject *sql,
 _sqlite3.Cursor.executemany as pysqlite_cursor_executemany
 
     sql: unicode
-    parameters: object = NULL
+    seq_of_parameters as parameters: object
```

Comment:
` as parameters` is not needed. Just rename the parameter.

Reply:
No, you'll need to change line 635 as well. You can't just rename the parameter without changing the parameter users as well.

Reply:
I'll of course change both lines if you prefer that to explicitly naming the parameter in AC.

Reply:
See feac1a121aee59797008e4264156bd8708d28b42